release-engineering/platforms.md

```diff
@@ -0,0 +1,65 @@
+# Supported Platforms
+
+The Kubernetes release process produces [artifacts](./artifacts.md) for
+different architectures and operating systems. We consider the combination of
+architecture (`GOARCH`) and operating system (`GOOS`) as "platforms". Target of
+this document is to outline different categories of platforms as well as guiding
+through their graduation criteria.
+
+## Tiers
+
+Support for different platforms are organized into three tiers, whereas each
+comes with a different set of guarantees.
+
+### Tier 1
+
+Tier 1 platforms are considered as "guaranteed to work". To achieve this, they
+have to fulfil the following criteria:
+
+- **All** official binary releases are provided for the platform. This includes
+  container images as well as deb and rpm packages. Building the artifacts is
+  integrated in the release process.
```

Comment:
I am wondering if maybe support levels should be "per-binary". For instance if we thoroughly test `kubectl` on `darwin-amd64` it would technically still fall into Tier 2 by this definition.

Reply:
Alright, I changed the doc to outline that we can scope Tiers to single binaries. Let me know what you think.